qtglean/CMakeLists.txt

```diff
@@ -122,10 +146,28 @@ if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
         DEPENDS
             ${CMAKE_CURRENT_BINARY_DIR}/glean/generated/metrics.rs
             ${CMAKE_CURRENT_BINARY_DIR}/glean/generated/pings.rs
+        SHARED ${SHARED}
     )
+
     target_link_libraries(qtglean PRIVATE qtglean_bindings)
 
-    set(HEADER_FILE qtglean.h)
+    if (ANDROID)
+        get_property(QTGLEAN_BINDINGS_LOCATION TARGET qtglean_bindings PROPERTY LOCATION_${CMAKE_BUILD_TYPE})
+        get_filename_component(QTGLEAN_BINDINGS_FILENAME ${QTGLEAN_BINDINGS_LOCATION} NAME)
+
+        # Move the qtglean shared library into the Android libs/ folder
+        add_custom_command(
+            TARGET qtglean
+            COMMAND ${CMAKE_COMMAND} -E copy
+                ${QTGLEAN_BINDINGS_LOCATION}
+                ${CMAKE_BINARY_DIR}/src/android-build/libs/${ANDROID_ABI}/${QTGLEAN_BINDINGS_FILENAME}
```

Comment:
Not a big fan of hard-coded paths. Did you consider adding it to: 

```set_property(TARGET mozillavpn PROPERTY QT_ANDROID_EXTRA_LIBS
    ${openssl_libs}/libcrypto_1_1.so
    ${openssl_libs}/libcrypto_1_1.so
    ${openssl_libs}/libssl_1_1.so)
    ${openssl_libs}/libssl_1_1.so
    ${openssl_libs}/libcrypto.so
    ${openssl_libs}/libssl.so
    ${QTGLEAN_BINDINGS_LOCATION}
    APPEND)
    ```

Reply:
I did consider it, I am not really sure why I landed on this though. I'll change it.